docs/applications/add-ssl-certificate.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Add an SSL Certificate to an Application
+intro: Add an SSL certificate to an application to allow HTTPS requests to it.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add and SSL Certificate to an Application
+1. On the choosen Devopness environment, click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click on the `DETAILS` button of the application you want to add an SSL certificate to
+1. Click on the `SSL` button (open lock icon)
```

Comment:
Please avoid prepositions and mentions to component type, as components might get replace (buttons by links, etc) and the docs would get outdated after any simple changes in the UI.

For a reference, see: 
1. https://github.com/devopness/devopness/blob/main/docs/ssh-keys/add-ssh-key.md
2. https://github.com/devopness/devopness/blob/main/docs/ssh-keys/remove-ssh-key.md

```suggestion
1. Click `DETAILS` on the application you want to add an SSL certificate to
1. On the upper-right corner click `SSL` (open lock icon)
```